treeherder/perf/secretary_tool.py

```diff
@@ -5,11 +5,13 @@
 from django.conf import settings as django_settings
 
 from treeherder.perf.models import BackfillRecord, BackfillReport, PerformanceSettings
+from treeherder.utils import default_serializer
 
 logger = logging.getLogger(__name__)
 
 
 # TODO: update the backfill status using data
+# TODO: consider making this a singleton
```

Comment:
Do you have bugs filed for these TODOs?

Reply:
For `# TODO: consider making this a singleton`, I filed [bug 1639112](https://bugzilla.mozilla.org/show_bug.cgi?id=1639112).